sklearn/linear_model/_ridge.py

```diff
@@ -40,6 +40,19 @@
 from ..utils.sparsefuncs import mean_variance_axis
 
 
+def _get_rescaled_operator(X, X_offset, sample_weight_sqrt):
+    """Helper function to get a LinearOperator for X @ coef."""
+
```

Comment:
can you expand the docstring to explain that it's useful to perform linear algebra with an implicitly centered sparse matrix